activities/codebase-stewardship/criteria-for-codebase-stewardship.md

```diff
@@ -54,21 +54,21 @@ The current owners of potential codebases are asked to answer these questions to
 
 ### Community potential
 
-1. Is there a 2 to 4 year roadmap?
+1. Is there a two to four year roadmap?
 2. Is there a public backlog in English or one you can make public?
-3. How far through your current set of goals are you?
-4. How realistic vs ambitious are your goals in the next 6 months?
-5. Is there a public bugtracking system in English or one you can make public?
+3. Is there a public bugtracking system in English or one you can make public?
```

Comment:
The original questions 3 and 4 were about the roadmap and backlog, not the bugtracking system. That's why they were asked in that order.

Reply:
aha! Then I would suggest making them either subquestions, or referring back to the backlog directly, to make this clearer

Reply:
I've put them back into the previous order. I wonder if the source of the confusion is the (deliberate) vagueness about 'your current set of goals'?